airflow/operators/sensors.py

```diff
@@ -52,10 +52,10 @@ def poke(self, context):
         raise AirflowException('Override me.')
 
     def execute(self, context):
-        started_at = datetime.now()
+        self.started_at = datetime.now()
```

Comment:
What is the reason to put this in self? It doesn't seem to be used anywhere else.